config/locales/de.yml

```diff
@@ -27,7 +26,7 @@ de:
         one: Recherche
         other: Recherchen
     hero:
-      title: Willkommen bei 100eyes. Nutze die Intelligenz der Community.
+      title: Willkommen bei %{project_name}. Nutze die Intelligenz der Community.
```

Comment:
The same applies here as above – the software editors are using is still "100eyes", only the project they are using 100eyes for has a name.